### ISSUE TITLE
Re-compilation failed

### ISSUE THREAD

**Issue by spencer627** [johnno1962/InjectionIII#324]
hello, can you help me to solve this problem

💉 ⚠️ Re-compilation failed (see: /var/folders/rz/1kjwv_5n03x5hq8spvdsz6nm0000gn/T/com.johnholdsworth.InjectionIII/command.sh)
fatal error: module 'Protobuf' was built in directory '/Users/shopping/Library/Developer/Xcode/DerivedData/FanXing-difkhhxbglbgwndoeaqtxhmaeptp/Build/Products/Debug-iphonesimulator/Protobuf/protobuf.framework' but now resides in directory '/Users/shopping/Library/Developer/Xcode/DerivedData/FanXing-difkhhxbglbgwndoeaqtxhmaeptp/Build/Products/Debug-iphonesimulator/Protobuf/Protobuf.framework'
note: imported by module 'FABData' in '/Users/shopping/Library/Developer/Xcode/DerivedData/ModuleCache.noindex/N35VOTMTPSKS/FABData-2WRAJC5G3X121.pcm'
note: imported by '/Users/shopping/Library/Developer/Xcode/DerivedData/FanXing-difkhhxbglbgwndoeaqtxhmaeptp/Build/Intermediates.noindex/PrecompiledHeaders/SharedPrecompiledHeaders/2060808295364420457/PrefixHeader.pch.gch'
1 error generated.

**Comment by johnno1962** [comment]
Hi, normally this type of problem is to do with case sensitivity. Is this the App Store version of InjectionIII? If so, can you try one of the non-sandboxed releases https://github.com/johnno1962/InjectionIII/releases ?

**Comment by Panway** [comment]
Thanks, download 2.8.1 from [release](https://github.com/johnno1962/InjectionIII/releases) resolve my problem.

Before(Mac App Store Version 2.8.0)：

```
💉 InjectionIII connected /Users/xxx/Downloads/GettingStarted/GettingStarted.xcodeproj
💉 Watching files under /Users/xxx/Downloads/GettingStarted
💉 ⚠️ Error reading /var/folders/16/y2lpd5211vv9svp066357p_h0000gn/T/com.johnholdsworth.InjectionIII/eval101.sh, scanCommand: /var/folders/16/y2lpd5211vv9svp066357p_h0000gn/T/com.johnholdsworth.InjectionIII/command.sh
```

After：
```
💉 Injected class 'GettingStarted.DetailViewController'
💉 Injected class 'GettingStarted.DetailViewController'
💉 As class DetailViewController has an @objc injected() method, InjectionIII will perform a "sweep" of live instances to determine which objects to message. If this fails, subscribe to the notification "INJECTION_BUNDLE_NOTIFICATION" instead.
💉 (note: notification may not arrive on the main thread)
====
```

**Comment by johnno1962** [comment]
👍

**Comment by spencer627** [comment]
oh, thx very much ,my Xcode is 12.3 and I find the version released with Xcode 12.3。it solve my problem。
I don't know why the version on APP Store fail？
